test/functional/Valhalla/src/org/openj9/test/lworld/ValhallaUtils.java

```diff
@@ -39,11 +39,11 @@ public class ValhallaUtils {
 	static final int ACC_DEFAULT = 0x1;
 	static final int ACC_NON_ATOMIC = 0x2;
 
-	final static class PreloadAttribute extends Attribute {
+	final static class LoadableDescriptorsAttribute extends Attribute {
```

Comment:
Please list modifiers in canonical order:
```java
	static final class LoadableDescriptorsAttribute extends Attribute {
```
Please do the same for other nested classes.